app/middlewares/rack/json_requests.rb

```diff
@@ -12,14 +12,22 @@ def initialize app
 
     def call env
       request = Rack::Request.new(env)
-      content_type = request.env['HTTP_ACCEPT']
-      return json_406_error if content_type.start_with?('application/json')
+      return json_406_error if invalid_api_request? request
 
       @app.call(env)
     end
 
     private
 
+    def invalid_api_request? request
+      return false if request.path.include?('collection_posts') ||
+                      request.path.include?('manifest.json') ||
+                      request.path.include?('feed.json')
```

Comment:
are there other paths?

Reply:
Not that I can think of. Bugsnag will tell us if we're wrong.